pkg/xds/context/mesh_context_builder.go

```diff
@@ -407,3 +407,7 @@ func (m *meshContextBuilder) resolveTLSReadiness(mesh *core_mesh.MeshResource, s
 	}
 	return tlsReady
 }
+
+func (m *meshContextBuilder) listOrdered(ctx context.Context, rl core_model.ResourceList, opts ...core_store.ListOptionsFunc) error {
+	return m.rm.List(ctx, rl, append(opts, core_store.ListOrdered())...)
+}
```

Comment:
I'd get rid of this method. You should be able to add core_store.ListOrdered in each calls anyway no?

Reply:
True, I can change it